elements_of_maps/elements_of_maps.md

```diff
@@ -180,8 +185,9 @@ In this section we will discuss other elements of maps, and learn about types of
 
 
 <div class = "warning">
+<b style="color: rgb(var(--color-highlight));">Warning!</b><br>
 
-Not everyone can distinguish colors. While colors can be a powerful tool, if colors are the only way your are distinguishing features on your map, those features will be invisible to some of your audience.
+Not everyone can distinguish colors. While colors can be a powerful tool, if colors are the only way you are distinguishing features on your map, those features will be invisible to some of your audience.
```

Comment:
While looking for ways to make this more accessible, I found a note on how to make heat maps more effective for colorblindness. Perhaps it would be useful to add a note on color blindness-friendly palettes and the addition of texture? There's tons of information on the Web Content Accessibility Guidelines 2018!

Reply:
Great idea! Do you have a link or two that we could include here? Like

> There are some color palettes that can be more effective for colorblindness and using texture instead of color can also make your maps more accessible. The WCAG 2018 has excellent resources on both color pallets (link) and textures (link).

Reply:
I'm looking for links right now! Will comment when I find it!

Reply:
> Great idea! Do you have a link or two that we could include here? Like
> 
> > There are some color palettes that can be more effective for colorblindness and using texture instead of color can also make your maps more accessible. The WCAG 2018 has excellent resources on both color pallets (link) and textures (link).

I'm looking for links right now! Will comment when I find it!

Reply:
Here are the links. These are all from the latest version, so I didn't need to link any from 2018.
https://www.w3.org/WAI/RD/wiki/Accessible_Maps#Tactile_Maps
https://www.w3.org/WAI/test-evaluate/preliminary/
https://www.w3.org/WAI/standards-guidelines/act/rules/afw4f7/proposed/